decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+require './person'
+class Decorator < Nameable
+    def initialize(nameable)
+        @nameable = Nameable
```

Comment:
- [ ] You should use the small letter `nameable`, instead of `Nameable` as it is causing errors.

  ```
  @nameable = nameable
  ```